pkg/backends/jmeter/backend.go

```diff
@@ -0,0 +1,280 @@
+		spec.WorkerConfig.Tag = b.workerConfig.Tag
+	}
+
+	return nil
+}
+
+// Sync check if Backend kubernetes resources have been create, if they have not been create them
+func (b *Backend) Sync(ctx context.Context, loadTest loadTestV1.LoadTest, reportURL string) error {
+	JMeterServices, err := b.kubeClientSet.CoreV1().Services(loadTest.Status.Namespace).List(ctx, metaV1.ListOptions{})
+	if err != nil {
+		return err
+	}
+
+	if len(JMeterServices.Items) == 0 {
+		_, err = b.kubeClientSet.CoreV1().ConfigMaps(loadTest.Status.Namespace).Create(ctx, b.NewConfigMap(loadTest), metaV1.CreateOptions{})
+		if err != nil && !kerrors.IsAlreadyExists(err) {
+			b.logger.Error("Error on creating testfile configmap", zap.Error(err))
+			return err
+		}
+
+		_, err = b.kubeClientSet.CoreV1().ConfigMaps(loadTest.Status.Namespace).Create(ctx, b.NewJMeterSettingsConfigMap(), metaV1.CreateOptions{})
+		if err != nil && !kerrors.IsAlreadyExists(err) {
+			return err
+		}
+
+		secret, err := b.NewSecret(loadTest)
+		if err != nil {
+			b.logger.Error("Error on creating environment variables secret", zap.Error(err))
+			return err
+		}
+
+		_, err = b.kubeClientSet.CoreV1().Secrets(loadTest.Status.Namespace).Create(ctx, secret, metaV1.CreateOptions{})
+		if err != nil && !kerrors.IsAlreadyExists(err) {
+			return err
+		}
+
+		configMaps, err := b.NewTestdataConfigMap(loadTest)
+		if err != nil {
+			b.logger.Error("Error on creating testdata configMaps", zap.Error(err))
+			return err
+		}
+
+		err = b.createPodsWithTestdata(ctx, configMaps, &loadTest, loadTest.Status.Namespace)
+		if err != nil {
+			return err
+		}
+
+		_, err = b.kubeClientSet.CoreV1().Services(loadTest.Status.Namespace).Create(ctx, b.NewJMeterService(), metaV1.CreateOptions{})
+		if err != nil && !kerrors.IsAlreadyExists(err) {
+			b.logger.Error("Error on creating new JMeter service", zap.Error(err))
+			return err
+		}
+
+		_, err = b.
+			kubeClientSet.
+			BatchV1().
+			Jobs(loadTest.Status.Namespace).
+			Create(
+				ctx,
+				b.NewJMeterMasterJob(loadTest, reportURL, b.podAnnotations),
+				metaV1.CreateOptions{},
+			)
+		if err != nil && !kerrors.IsAlreadyExists(err) {
+			b.logger.Error("Error on creating new JMeter master Job", zap.Error(err))
+			return err
+		}
+
+		b.logger.Info(
+			"Created JMeter resources",
+			zap.String("LoadTest", loadTest.GetName()),
+			zap.String("namespace", loadTest.Status.Namespace),
+		)
+	}
+	return nil
+}
+
+// SyncStatus check the Backend resources and calculate the current status of the LoadTest from them
+func (b *Backend) SyncStatus(ctx context.Context, loadTest loadTestV1.LoadTest, loadTestStatus *loadTestV1.LoadTestStatus) error {
+	// Get the Namespace resource
+	namespace, err := b.namespaceLister.Get(loadTestStatus.Namespace)
+	if err != nil {
+		// The LoadTest resource may no longer exist, in which case we stop
+		// processing.
+		if kerrors.IsNotFound(err) {
+			loadTestStatus.Phase = loadTestV1.LoadTestFinished
+			return nil
+		}
```

Comment:
Are we missing an err return?
```suggestion
		}
		return err
```